src/averageAges.js

```diff
@@ -37,7 +40,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const newWomanArray = withChildren
+    ? people.filter(person => people
+      .find(human => human.mother === person.name))
+    : people.filter(person => person.sex === 'f');
```

Comment:
But you duplicated usage of filter method. Let's follow DRY and move ternary operator inside filter method to just select needed condition to filter